arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -248,11 +248,7 @@
                 },
                 "properties":
                 {
-                    "allOf": [
```

Comment:
I believe allOf in line https://github.com/Azure/azure-rest-api-specs/pull/1005#issuecomment-286594317 needs to come out of "properties", so swagger doesn't think "allOf" is the name of a property.